tests/lvmo/test_lvm_clone_base.py

```diff
@@ -42,9 +42,10 @@ class TestLvmSnapshot(ManageTest):
 
     """
 
-    ocp_version = get_ocp_version()
-    pvc_size = 100
-    access_mode = constants.ACCESS_MODE_RWO
+    def __init__(self):
+        self.ocp_version = get_ocp_version()
```

Comment:
can we remove this? didn't its usage in test cases. same in other files as well

Reply:
I am not removing those, just initiate those volumes in constructor and not on class level which from some reason causing such issues:
https://url.corp.redhat.com/ea867a8
```
2023-04-11 17:22:42          char = seperator if seperator else "."
2023-04-11 17:22:42          if config.ENV_DATA.get("skip_ocp_deployment"):
2023-04-11 17:22:42  >           raw_version = json.loads(run_cmd("oc version -o json"))["openshiftVersion"]
2023-04-11 17:22:42  E           KeyError: 'openshiftVersion'
2023-04-11 17:22:42  
2023-04-11 17:22:42  ocs_ci/utility/utils.py:1838: KeyError
2023-04-11 17:22:42  =============================== warnings summary ===============================
2023-04-11 17:22:42  tests/lvmo/test_lvm_clone_base.py:39
2023-04-11 17:22:42    /home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/tests/lvmo/test_lvm_clone_base.py:39: PytestCollectionWarning: cannot collect test class 'TestLvmSnapshot' because it has a __init__ constructor (from: tests/lvmo/test_lvm_clone_base.py)
2023-04-11 17:22:42      class TestLvmSnapshot(ManageTest):
2023-04-11 17:22:42  
2023-04-11 17:22:42  tests/lvmo/test_lvm_multi_clone.py:42
2023-04-11 17:22:42    /home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/tests/lvmo/test_lvm_multi_clone.py:42: PytestCollectionWarning: cannot collect test class 'TestLvmMultiClone' because it has a __init__ constructor (from: tests/lvmo/test_lvm_multi_clone.py)
2023-04-11 17:22:42      class TestLvmMultiClone(ManageTest):
2023-04-11 17:22:42  
2023-04-11 17:22:42  tests/lvmo/test_lvm_multi_snapshot.py:42
2023-04-11 17:22:42    /home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/tests/lvmo/test_lvm_multi_snapshot.py:42: PytestCollectionWarning: cannot collect test class 'TestLvmMultiSnapshot' because it has a __init__ constructor (from: tests/lvmo/test_lvm_multi_snapshot.py)
2023-04-11 17:22:42      class TestLvmMultiSnapshot(ManageTest):
2023-04-11 17:22:42  
2023-04-11 17:22:42  tests/lvmo/test_lvm_snapshot_base.py:39
2023-04-11 17:22:42    /home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/tests/lvmo/test_lvm_snapshot_base.py:39: PytestCollectionWarning: cannot collect test class 'TestLvmSnapshot' because it has a __init__ constructor (from: tests/lvmo/test_lvm_snapshot_base.py)
2023-04-11 17:22:42      class TestLvmSnapshot(ManageTest):
```

But looks like to move it to constructor didn't help. I have now cluster paused to debug the issue.